### ISSUE TITLE
dotnet add package output is not optimal 

### ISSUE THREAD

**Issue by anangaur** [NuGet/Home#8607]
Issues:
1. Too verbose
2. Not color scheme to indicate failure or warning

Example of verbosity of the output for a successful install:
```
C:\Users\anangaur\play\tmp\tmpsoln
λ dotnet add package my.sample.lib
  Writing C:\Users\anangaur\AppData\Local\Temp\tmp6A43.tmp
info : Adding PackageReference for package 'my.sample.lib' into project 'C:\Users\anangaur\play\tmp\tmpsoln\tmpsoln.csproj'.
info : Restoring packages for C:\Users\anangaur\play\tmp\tmpsoln\tmpsoln.csproj...
info :   GET https://nuget.pkg.github.com/anangaur/download/my.sample.lib/index.json
info :   GET https://api.nuget.org/v3-flatcontainer/my.sample.lib/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/my.sample.lib/index.json 216ms
info :   OK https://nuget.pkg.github.com/anangaur/download/my.sample.lib/index.json 769ms
info :   GET https://api.nuget.org/v3-flatcontainer/my.sample.lib/6.0.1/my.sample.lib.6.0.1.nupkg
info :   OK https://api.nuget.org/v3-flatcontainer/my.sample.lib/6.0.1/my.sample.lib.6.0.1.nupkg 208ms
info :   GET https://api.nuget.org/v3-flatcontainer/contentfilesexample/index.json
info :   GET https://nuget.pkg.github.com/anangaur/download/contentfilesexample/index.json
info :   GET https://api.nuget.org/v3-flatcontainer/nunit/index.json
info :   GET https://nuget.pkg.github.com/anangaur/download/nunit/index.json
info :   GET https://api.nuget.org/v3-flatcontainer/newtonsoft.json/index.json
info :   GET https://nuget.pkg.github.com/anangaur/download/newtonsoft.json/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/newtonsoft.json/index.json 91ms
info :   GET https://api.nuget.org/v3-flatcontainer/newtonsoft.json/10.0.2/newtonsoft.json.10.0.2.nupkg
info :   OK https://api.nuget.org/v3-flatcontainer/newtonsoft.json/10.0.2/newtonsoft.json.10.0.2.nupkg 9ms
info :   OK https://api.nuget.org/v3-flatcontainer/contentfilesexample/index.json 246ms
info :   GET https://api.nuget.org/v3-flatcontainer/contentfilesexample/1.0.2/contentfilesexample.1.0.2.nupkg
info :   OK https://api.nuget.org/v3-flatcontainer/nunit/index.json 248ms
info :   OK https://api.nuget.org/v3-flatcontainer/contentfilesexample/1.0.2/contentfilesexample.1.0.2.nupkg 9ms
info :   GET https://api.nuget.org/v3-flatcontainer/nunit/3.8.1/nunit.3.8.1.nupkg
info :   OK https://api.nuget.org/v3-flatcontainer/nunit/3.8.1/nunit.3.8.1.nupkg 10ms
info :   NotFound https://nuget.pkg.github.com/anangaur/download/contentfilesexample/index.json 378ms
info :   NotFound https://nuget.pkg.github.com/anangaur/download/nunit/index.json 582ms
info :   NotFound https://nuget.pkg.github.com/anangaur/download/newtonsoft.json/index.json 586ms
info : Installing Newtonsoft.Json 10.0.2.
info : Installing NUnit 3.8.1.
info : Installing ContentFilesExample 1.0.2.
info : Installing My.Sample.Lib 6.0.1.
info : Package 'my.sample.lib' is compatible with all the specified frameworks in project 'C:\Users\anangaur\play\tmp\tmpsoln\tmpsoln.csproj'.
info : PackageReference for package 'my.sample.lib' version '6.0.1' added to file 'C:\Users\anangaur\play\tmp\tmpsoln\tmpsoln.csproj'.
info : Committing restore...
info : Generating MSBuild file C:\Users\anangaur\play\tmp\tmpsoln\obj\tmpsoln.csproj.nuget.g.props.
info : Writing assets file to disk. Path: C:\Users\anangaur\play\tmp\tmpsoln\obj\project.assets.json
log  : Restore completed in 4.73 sec for C:\Users\anangaur\play\tmp\tmpsoln\tmpsoln.csproj.
```

For a failed install, it doesn;t indicate by color mechanism if anything failed unless I read the output. Eg.:
![image](https://user-images.githubusercontent.com/14800916/65181120-32060580-da13-11e9-994c-d12d220b3bc0.png)

**Comment by nkolev92** [comment]
Linking the dotnet nuget verbosity issue as per our conversation yesterday @anangaur 
https://github.com/NuGet/Home/issues/6374
